plugins/woocommerce/src/Blocks/BlockTypes/ProductGallery.php

```diff
@@ -186,8 +186,8 @@ protected function render( $attributes, $content, $block ) {
 				'data-wc-context',
 				wp_json_encode(
 					array(
-						'selectedImage'                   => $product_gallery_first_image_id,
 						'firstMainImageId'                => $product_gallery_first_image_id,
+						'selectedImageIndex'              => 1,
```

Comment:
I guess this is why we mix zero/one based indexing. Any danger is defaulting to `0`?